Tools/serial/rc.serial.jinja

```diff
@@ -0,0 +1,35 @@
+{# jinja template to generate the serial autostart script. #}
+
+# serial autostart script generated with generate_serial_config.py
+
+
+# make sure all baudrates are marked as used
+param touch
+{%- for serial_device in serial_devices %} SER_{{ serial_device.tag }}_BAUD
```

Comment:
This could be skipped so that it's only visible to the user when on an appropriate config.
For example frsky telem, iridum, syslink, etc shouldn't present a configurable baudrate.

Reply:
Yes and they'll still be there if you have anything (Mavlink, GPS, etc) enabled on that port. In the cases where it's not used we can hide it, skip the sync, and close one more little hole where users are able to try doing something that doesn't make any sense. Since all of these parameters set via command line are reboot_required I don't really see the problem (set mavlink, reboot, serial mavlink baudrate param available).

This is a case where handling the param in the module itself could actually be a lot better as it would enable runtime changes.